src/cascadia/TerminalApp/TerminalTab.cpp

```diff
@@ -383,7 +390,7 @@ namespace winrt::TerminalApp::implementation
             return RS_(L"MultiplePanes");
         }
         const auto activeContent = GetActiveContent();
-        return activeContent ? activeContent.Title() : L"";
+        return activeContent ? activeContent.Title() : winrt::hstring{ L"" };

```

Comment:
nit

```suggestion
        return activeContent ? activeContent.Title() : winrt::hstring{};
```

These are equivalent, but now that you have to specify the type explicitly we may as well switch to the nullary constructor